manifests/server.pp

```diff
@@ -81,6 +74,16 @@
     notify  => Service['puppetserver'],
   }
 
+  file { 'puppetserver_ca_cfg':
+    ensure  => 'file',
+    path    => '/etc/puppetlabs/puppetserver/services.d/ca.cfg',
```

Comment:
Only if you want to support older puppetserver versions the path needs to be parameterized.
See https://github.com/voxpupuli/puppet-puppetserver/issues/52 for details.